stix_shifter_modules/secretserver/stix_translation/json/event_from_stix_map.json

```diff
@@ -14,28 +14,21 @@
     }
   },
 
-  "user-account": {
```

Comment:
Why remove this and stick it in a custom object? When possible, standard STIX objects should be used.

Reply:
Done the changes